app/views/volunteer_ops/show.html.erb

```diff
@@ -6,7 +6,12 @@
   <p>
     <%= simple_format auto_link @volunteer_op.description %>
   </p>
-
+  <% if has_diff_location? @volunteer_op %>
```

Comment:
I'm thinking maybe simpler to just show these fields all the time? i.e. not worry about whether we've specified that there is a different address or not ...